coffee-chats/src/main/java/com/google/step/coffee/entity/DateRange.java

```diff
@@ -165,7 +165,9 @@ public String toString() {
     if (this.overlaps(removeRange)) {
       if (this.getStart().before(removeRange.getStart())) {
         splitRanges.add(new DateRange(this.getStart(), removeRange.getStart()));
-      } else if (removeRange.getEnd().before(this.getEnd())) {
+      }
+
+      if (removeRange.getEnd().before(this.getEnd())) {
```

Comment:
I see what you were referring to now, it should have been a separate if not an else if. Thanks for spotting.